src/react-styleguide/.gitignore

```diff
@@ -14,3 +14,4 @@ build
 .env
 npm-debug.log
 src/assets/* # Clone from vanilla html version
+.eslint.js
```

Comment:
If there are linting rules being added they should probably be committed for anyone else who contributes. When did this project get ejected from CRA `react-scripts`?